stable/redmine/README.md

```diff
@@ -61,6 +61,7 @@ The following tables lists the configurable parameters of the Redmine chart and
 | `redminePassword` | Application password | _random 10 character long alphanumeric string_ |
 | `redmineEmail` | Admin email | `user@example.com` |
 | `redmineLanguage` | Redmine default data language | `en` |
+| `redmineEnvironment` | Environment variables, passed to redmine| `[]` |
```

Comment:
@flah00 I think it would make sense semantically to rename the template value something like `extraVars`, since there are already other vars specified. See https://github.com/kubernetes/charts/commit/3850b04734b900a50225aefd3575808f39c0fe4f for an example. I would love if we could introduce this with consistency across applicable chart containers.